src/main/java/ro/itschool/ema/models/dtos/OrganizerDTO.java

```diff
@@ -0,0 +1,33 @@
+package ro.itschool.ema.models.dtos;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Past;
+import lombok.Data;
+import org.springframework.validation.annotation.Validated;
+
+import java.io.Serializable;
+import java.util.Date;
+
+@Data
+@Validated
+public class OrganizerDTO implements Serializable {
+    private Long id;
+
+    @NotNull(message = "Field cannot be null.")
+    private String firstName;
+
+    @NotNull(message = "Field cannot be null.")
+    private String lastName;
+
+    @NotNull(message = "Field cannot be null.")
+    @Past(message = "Invalid date.")
+    private Date dateOfBirth;
```

Comment:
if we care only about the date of birth, without the time, we could use LocalDate